start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
There is problem with that approach: http://stackoverflow.com/questions/27959860/how-to-merge-host-folder-with-container-folder-in-docker/27960880#27960880

MediaWiki allows us to install some default extensions. So, the extensions folder may not be empty initially.